lib/python/rosie/db_create.py

```diff
@@ -185,9 +201,21 @@ def load(self, repos_path):
                     "\r%s... loading revision %d of %d" %
                     (Reporter.PREFIX_INFO, revision, youngest))
                 sys.stdout.flush()
-            self.post_commit_hook.run(
-                repos_path, str(revision), no_notification=True)
-            event = RosieDatabaseLoadEvent(repos_path, revision, youngest)
+            try:
+                self.post_commit_hook.run(
+                    repos_path, str(revision), no_notification=True)
+            except (ConfigSyntaxError, DatabaseError, InfoFileError) as err:
+                if sys.stdout.isatty():
+                    sys.stdout.write("\r")
+                    sys.stdout.flush()
+                err_msg = "Exception occurred: {0} - {1}".format(
+                    type(err).__name__, str(err))
+                message = ("Could not load revision {0} of {1} as the post-"
+                           "commit hook failed:\r{2}\r".format(
```

Comment:
`\r`?

Did you mean `\n`?

`\r` is a carriage return without the resulting newline, it overwrites what comes before it:

```console
$ python2 -c 'print "foo\rbar\r"'
bar
```